src/Native/Unix/System.Security.Cryptography.Native/opensslshim.h

```diff
@@ -168,10 +168,16 @@ void SSL_get0_alpn_selected(const SSL* ssl, const unsigned char** protocol, unsi
     PER_FUNCTION_BLOCK(ERR_reason_error_string, true) \
     PER_FUNCTION_BLOCK(EVP_aes_128_cbc, true) \
     PER_FUNCTION_BLOCK(EVP_aes_128_ecb, true) \
+    PER_FUNCTION_BLOCK(EVP_aes_128_gcm, true) \
+    PER_FUNCTION_BLOCK(EVP_aes_128_ccm, true) \
```

Comment:
You marked both of these as required (the `true`) but probed for them with API_EXISTS.  That seems a bit inconsistent.

Reply:
I followed the pattern, didn't have a clue that's what the true stood for - will change